evm_arithmetization/src/generation/prover_input.rs

```diff
@@ -396,7 +396,7 @@ impl<F: Field> GenerationState<F> {
     /// Simulate the user's code and store all the jump addresses with their
     /// respective contexts.
     fn generate_jumpdest_table(&mut self) -> Result<(), ProgramError> {
-        let checkpoint = self.checkpoint();
+        let _checkpoint = self.checkpoint();
```

Comment:
As this is unused, why not removing it? It was necessary when we were doing the actual JD execution, but now that's its simulated, we can scrap it out.